torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6972,6 +6985,25 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            safe_casts_outputs=True,
            supports_forward_ad=True,
            sample_inputs_func=sample_inputs_xlog1py),
+    OpInfo('special.zeta',
+           aten_name='special_zeta',
+           dtypes=all_types_and(torch.bool),
+           supports_autograd=False,
+           safe_casts_outputs=True,
+           sample_inputs_func=sample_inputs_binary_pwise),
+    # OpInfo entry to verify the gradient formula of `other`/`q`
```

Comment:
Smart.

One goal of our "writing ops in Python" project (which we're looking at for H2 2021) is to simplify supporting scalar x tensor and tensor x scalar operations, possibly by automating the generation of those functions